tests/lib/fakestore/cmd/fakestore/cmd_make_refreshable.go

```diff
@@ -0,0 +1,43 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"github.com/snapcore/snapd/tests/lib/fakestore/refresh"
+)
+
+type cmdMakeRefreshable struct {
+	TopDir string `long:"dir" description:"Directory to be used by the store to keep and serve snaps, <dir>/asserts is used for assertions"`
+}
+
+func (x *cmdMakeRefreshable) Execute(args []string) error {
+	return runManage(x.TopDir, args)
+}
+
+var shortMakeRefreshableHelp = "List of snaps with new versions"
+
+func init() {
+	parser.AddCommand("make-refreshable", shortMakeRefreshableHelp, "", &cmdMakeRefreshable{})
+}
+
+func runManage(topDir string, snaps []string) error {
```

Comment:
not sure having this helper is useful now, Execute could just call the function directly